### PR TITLE
Vet clinic database: add "join table" for visits

### PR DESCRIPTION
**Pull request by CindyDorantes**
## Project requirements   :bookmark_tabs:

- Create a table named `vets` with the following columns:
  - `id`: integer (set it as autoincremented PRIMARY KEY)
  - `name`: string
  - `age`: integer
  - `date_of_graduation`: date
- There is a **many-to-many** relationship between the tables `species` and `vets`: a vet can specialize in multiple species, and a species can have multiple vets specialized in it. Create a "join table" called `specializations` to handle this relationship.
- There is a **many-to-many** relationship between the tables `animals` and `vets`: an animal can visit multiple vets and one vet can be visited by multiple animals. Create a "join table" called `visits` to handle this relationship, it should also keep track of the date of the visit.
- Insert the data provided for vets.
- Insert the data provided for specializations.
- Insert the data provided for visits.
- Write queries to answer the questions provided

## Screenshots  :camera:

query_animals_13

query_animals_14

query_animals_15

query_animals_16

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -150,4 +152,74 @@ FROM owners O
...
+LEFT JOIN specializations SZ ON V.id = SZ.vet_id
+LEFT JOIN species S ON SZ.species_id = S.id;
+-- List all animals that visited Stephanie Mendez between 
+-- April 1st and August 30th, 2020.
+SELECT A.name
+FROM animals A
+JOIN visits VS ON A.id = VS.animal_id
+JOIN vets V ON VS.vet_id = V.id
+WHERE V.name = 'Stephanie Mendez'
+GROUP BY A.name;
```
Comment (ShahierNashaat) [inline queries.sql]: - [x] Kindly specify the date where the visit happened as required.

```sql
SELECT animals.name, vistis.visit_date
  FROM visits
  LEFT JOIN animals ON animals.id = visits.animal_id
  LEFT JOIN vets ON vets.id = visits.vet_id
  WHERE vets.name = 'Stephanie Mendez'
  AND visit_date > '2020-04-01' AND visit_date < '2020-08-30'

```

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @CindyDorantes, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Your code is clean and well documented ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (CindyDorantes) [reply queries.sql]: Done! Thanks for the feedback

**Review by prolajumokeoni — APPROVED** [review]
Hi @CindyDorantes, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**